Update bootstrap documentation (#673)
digital-asset/docs.daml.com: docs/2.7.9/docs/canton/usermanual/identity_management.rst

~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> There are situations where a node should not be automatically initialized, but where we prefer to control

Reviewer: ```suggestion
There are situations where a node should not be automatically initialized, but where you should control
```

> each step of the initialization. For example, when a node in the setup does

Reviewer: ```suggestion
each step of the initialization. For example, this might be the case when a node in the setup does
```

not control its own identity, or when you do not want to store the identity key on the node for security
reasons.

> In the following, we demonstrate the basic steps how to initialise a node:

Reviewer: ```suggestion
The following demonstrates the basic steps on how to initialize a node:
```
